articles/aks/deploy-confidential-containers-default-policy.md

```diff
@@ -377,19 +377,19 @@ For this preview release, we recommend for test and evaluation purposes to eithe
     kubectl get svc consumer -n kafka 
     ```
 
-Copy and paste the external IP address of the consumer service into your browser and observe the decrypted message. 
+1. Copy and paste the external IP address of the consumer service into your browser and observe the decrypted message. 
 
-The following resemblers the output of the command:
-
-```output
-Welcome to Confidential Containers on AKS!
-Encrypted Kafka Message: 
-Msg 1: Azure Confidential Computing
-```
+    The following resembles the output of the command:
```

Comment:
@MGoedtel Thank you for reviewing. I have made the proper change based on your suggestion.